safe_transaction_service/history/models.py

```diff
@@ -497,7 +497,11 @@ class Meta:
             Index(fields=["_from", "timestamp"]),
             Index(fields=["to", "timestamp"]),
         ]
-        unique_together = (("ethereum_tx", "log_index"),)
+        constraints = [
+            models.UniqueConstraint(
+                fields=["ethereum_tx", "log_index"], name="unique_token_transfer_index"
```

Comment:
Would this work if you don't set `name`? I would prefer to use the default django naming if posible

Reply:
It seems that the 'name' parameter is mandatory:
![Screenshot 2024-02-27 at 17 36 26](https://github.com/safe-global/safe-transaction-service/assets/6717781/6e9e5dfa-3c7a-447a-90c2-53f961425f24)

Reply:
Thanks Felipe! Makes sense then. I don't understand why they set an optional field for mandatory argument, but... 🤷🏻